gadget/gadget.go

```diff
@@ -45,10 +45,11 @@ const (
 	// GPT identifies a GUID Partition Table partitioning schema
 	GPT = "gpt"
 
-	SystemBoot = "system-boot"
-	SystemData = "system-data"
-	BootImage  = "bootimg"
-	BootSelect = "bootselect"
+	SystemBoot     = "system-boot"
+	SystemData     = "system-data"
+	SystemRecovery = "system-recovery"
```

Comment:
Maybe a "TODO"/"FIXME" here like: `// clarify (with Gustavo?) that this is really the final name for this new role`

Reply:
Will add a FIXME, I suspect this will be system-seed to keep consistency but let's confirm that.